devicon.json

```diff
@@ -1221,6 +1221,33 @@
         "color": "#0080FF",
         "aliases": []
     },
+    {
+        "name": "discordjs",
+        "tags": [
+            "wrapper",
+            "api_wrapper",
+            "nodejs"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#2a2c3e",
+        "aliases": [
+          {
+              "base": "plain",
+              "alias": "original"
+          }
```

Comment:
```suggestion
```
One final change: since we now have the `plain` SVG, we should just remove the alias section